linked_list/linked_list.py

```diff
@@ -0,0 +1,195 @@
+from __future__ import print_function  # For Python 2
+
+
+class node(object):
+    def __init__(self, data=None, next_=None):
+        # Initialises Node
+        self.data = data
+        self.next = next_
+
+    def __str__(self):
+        return str(self.data)
+
+
+class linked_list(object):
+    def __init__(self):
+        # Initialises Linked List.
+        self.head = None
+        self.size = 0
+
+    def add_front(self, data):
+        # Inserts element at first position.
+        n = node(data)
+        n.next = self.head
+        self.head = n
+        self.size += 1
+
+    def add_last(self, data):
+        # Inserts element at last position.
+        if not self.head:
+            n = node(data)
+            self.head = n
+            self.size = 1
+            return
+        else:
+            n = self.head
+            while n.next is not None:
+                n = n.next
+            new_node = node(data)
+            n.next = new_node
+            self.size += 1
+            return
+
+    def add(self, data, index):
```

Comment:
I didn't say `forget` I said if he decides and at the same moment they are lazy enough to use `add_front` or `add_back`. I think here then it would be no harm to use `index-self.size`. It will be just another feature for user 😉